resources/puppet-webhook.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+After=network.target
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/puppet-webhook
```

Comment:
should this service run as root?

Reply:
I simply used Puppet as a template and it runs as root. I think that will be required since all the Puppet directories are owned by root by default and this will be modifying files in those directories. I was unable to get it to work with any user other than root. :/

Reply:
okay. let's keep that for now and have a look at it later on.